docs/sources/flow/reference/components/pyroscope.ebpf.md

```diff
@@ -192,18 +192,47 @@ discovery.kubernetes "all_pods" {
     field = "spec.nodeName=" + env("HOSTNAME")
     role = "pod"
   }
-
 }
 
 discovery.relabel "local_pods" {
   targets = discovery.kubernetes.all_pods.targets
+  rule {
+    action = "drop"
+    regex = "Succeeded|Failed"
+    source_labels = ["__meta_kubernetes_pod_phase"]
+  }
   rule {
     action = "replace"
-    replacement = "${1}/${2}"
-    separator = "/"
+    regex = "(.*)@(.*)"
+    replacement = "ebpf/${1}/${2}"
+    separator = "@"
```

Comment:
Not quite sure why you switched to `@`, but I guess that will not surface anywhere else

Reply:
I just copy pasted it from our jsonnet, and it was probably copypasted to the jsonnet from somewhere else

Reply:
<img width="709" alt="Screenshot 2023-09-06 at 8 17 19 AM" src="https://github.com/grafana/agent/assets/331773/64f68411-c28c-4521-b5b7-d1e2366ec12a">

https://grafana.com/blog/2022/03/21/how-relabeling-in-prometheus-works/